streams/src/main/java/org/apache/kafka/streams/processor/internals/ChangelogReader.java

```diff
@@ -26,24 +26,18 @@
 public interface ChangelogReader extends ChangelogRegister {
     /**
      * Restore all registered state stores by reading from their changelogs
+     *
+     * @return number of records restored in this call
      */
-    void restore();
-
-    /**
-     * Transit to restore active changelogs mode
-     */
-    void enforceRestoreActive();
```

Comment:
With the restore changelog reader solely owned by a single thread, we do not need make this public APIs any more.